packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -427,10 +434,6 @@ pub struct ExperimentalConfig {
     pub client_router_filter_allowed_rate: Option<f64>,
     pub client_router_filter_redirects: Option<bool>,
     pub fetch_cache_key_prefix: Option<String>,
-    /// In-memory cache size in bytes.
-    ///
-    /// If `isr_memory_cache_size: 0` disables in-memory caching.
-    pub isr_memory_cache_size: Option<f64>,
```

Comment:
Did we leave a warning when the experimental configs are used still?

Reply:
Will add later in a different one